conftest.py

```diff
@@ -64,20 +64,33 @@ def pytest_runtest_makereport(item, call):
     setattr(item, "rep_" + rep.when, rep)
 
 
-@pytest.fixture(autouse=True)
-def get_task_logs_on_failure(request):
-    """ Scheduler should be the only task running as root
-    """
-    yield
+def get_rotating_task_log_lines(task_id: str, task_file: str):
+    rotated_filenames = [task_file,]
+    rotated_filenames.extend(['{}.{}'.format(task_file, i) for i in range(1, 10)])
```

Comment:
Just a thought, isn't this going to append the logs out of order? The file with no enumeration contains the most recent logs, but this is being returned first, followed by the oldest(?) logs?